### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by markpfaff**

### CODE REVIEW

File: fall-2017/mark_pfaff/swagger/swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+        - in: body
+          name: body
+          description: User object that needs to be added
+          required: true
+          schema:
+            $ref: '#/definitions/User'
+      responses:
+        '200':
+          description: Success!
+        '404':
```
Comment (toddysm) [inline fall-2017/mark_pfaff/swagger/swagger.yaml]: 404 is not appropriate Response for the POST

File: fall-2017/mark_pfaff/swagger/swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+    put:
+      tags:
+        - user
+      summary: Update user
+      description: ''
+      operationId: updateUser
+      produces:
+        - application/json
+      parameters:
+        - in: body
```
Comment (toddysm) [inline fall-2017/mark_pfaff/swagger/swagger.yaml]: You need also the userId parameter here

File: fall-2017/mark_pfaff/swagger/swagger.yaml [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+          name: body
+          description: name that need to be updated
+          required: true
+          schema:
+            $ref: '#/definitions/User'
+      responses:
```
Comment (toddysm) [inline fall-2017/mark_pfaff/swagger/swagger.yaml]: There is also 200 response if everything id OK

File: fall-2017/mark_pfaff/swagger/swagger.yaml [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+      produces:
+        - application/json
+      parameters:
+        - name: userid
+          in: path
+          description: User id to delete
+          required: true
+          type: integer
+          format: int64
+      responses:
```
Comment (toddysm) [inline fall-2017/mark_pfaff/swagger/swagger.yaml]: You need also 200 response if everything is OK

File: fall-2017/mark_pfaff/swagger/swagger.yaml [hunk 5]
```diff
@@ -0,0 +1,305 @@
...
+      parameters:
+        - name: username
+          in: query
+          description: Find user by name
+          required: true
+          type: string
+      responses:
+        '200':
+          description: successful operation
+          schema:
```
Comment (toddysm) [inline fall-2017/mark_pfaff/swagger/swagger.yaml]: The response is an array of users

File: fall-2017/mark_pfaff/swagger/swagger.yaml [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+      produces:
+        - application/json
+      parameters:
+        - name: userid
+          in: path
+          description: ID of user task that needs to be updated
+          required: true
+          type: integer
+          format: int64
+        - name: name
```
Comment (toddysm) [inline fall-2017/mark_pfaff/swagger/swagger.yaml]: We don't use form data for each piece of data but a User JSON object

File: fall-2017/mark_pfaff/swagger/swagger.yaml [hunk 7]
```diff
@@ -0,0 +1,305 @@
...
+          required: true
+          type: integer
+          maximum: 10
+          minimum: 1
+          format: int64
+      responses:
+        '200':
+          description: successful operation
+          schema:
+            $ref: '#/definitions/User'
```
Comment (toddysm) [inline fall-2017/mark_pfaff/swagger/swagger.yaml]: It should return Task object

File: fall-2017/mark_pfaff/swagger/swagger.yaml [hunk 8]
```diff
@@ -0,0 +1,305 @@
...
+          format: int64
+        - name: userid
+          in: path
+          description: ID of the user of the task that needs to be deleted
+          required: true
+          type: integer
+          minimum: 1
+          format: int64
+      responses:
+        '400':
```
Comment (toddysm) [inline fall-2017/mark_pfaff/swagger/swagger.yaml]: 200 response

File: fall-2017/mark_pfaff/swagger/swagger.yaml [hunk 9]
```diff
@@ -0,0 +1,305 @@
...
+        - name: status
+          in: formData
+          description: Updated status of the pet
+          required: false
+          type: string
+      responses:
+        '405':
+          description: Invalid input
+  '/users/{userid}/tasks/{taskid}':
+    get:
```
Comment (toddysm) [inline fall-2017/mark_pfaff/swagger/swagger.yaml]: I think you are missing a PUT here

**Comment by markpfaff** [comment]
@toddysm latest commit updated with requested changes

**Comment by toddysm** [comment]
@markpfaff I don't see the changes in the PR - you need to update the PR to see the new commit.

**Pull request merged**